spec/rspec_puppet_facts_spec.rb

```diff
@@ -862,7 +861,7 @@
 
       it 'returns CentOS facts from a facter version matching 3.8' do
         is_expected.to include(
-          'centos-7-x86_64' => include(:facterversion => '3.8.0'),
+          'centos-7-x86_64' => include(:facterversion => '3.9.6'),
```

Comment:
On IRC we had some discussion about this. Some argued that if you request `3.9.5` (as can be seen on line 851) that it should return 3.8.0 instead of 3.9.6, like it did previously.

It can also be argued that 3.9.6 is closer. Please discuss this further so I know what to implement.

Either way, I need to either change the description of this expectation or this expectation itself.

Reply:
I've found a way to modify the response to make sure it falls back properly.